src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/action/ViewCommentAction.kt

```diff
@@ -0,0 +1,36 @@
+package com.atlassian.performance.tools.jiraactions.api.action
+
+import com.atlassian.performance.tools.jiraactions.api.VIEW_COMMENT
+import com.atlassian.performance.tools.jiraactions.api.WebJira
+import com.atlassian.performance.tools.jiraactions.api.measure.ActionMeter
+import com.atlassian.performance.tools.jiraactions.api.memories.CommentMemory
+import org.apache.logging.log4j.LogManager
+import org.apache.logging.log4j.Logger
+import javax.json.Json
+
+class ViewCommentAction(
+    private val jira: WebJira,
+    private val meter: ActionMeter,
+    private val commentMemory: CommentMemory
+) : Action {
+    private val logger: Logger = LogManager.getLogger(this::class.java)
+
+    override fun run() {
+        val comment = commentMemory.recall()
+        if (comment == null) {
+            logger.debug("Skipping View Comment action. I have no knowledge of comments.")
+            return
+        }
+
+        meter.measure(
+            key = VIEW_COMMENT,
+            action = { jira.goToComment(comment.url).validateCommentIsFocused(comment.id) },
+            observation = { page ->
+                Json.createObjectBuilder()
+                    .add("issueKey", page.getIssueKey())
+                    .add("commentId", comment.id)
+                    .build()
+            }
```

Comment:
Do we have a use case for those observations?

Reply:
https://github.com/atlassian/jira-actions/pull/103#discussion_r864589103

Without issue key we won't be able to know if the view comment action performance depends on the issue viewed. Some issues may have more comments than others and some even can have content of those comments that could render longer. I think it's worth to add.

Reply:
@dagguh is the use case acceptable?